src/utils/offlineRequestHandler.ts

```diff
@@ -295,24 +295,34 @@ export class OfflineRequestHandler {
         SerializedRequest[]
       >((resolve) => {
         const requests: SerializedRequest[] = [];
+        const noFormKeyRequests: SerializedRequest[] = [];
```

Comment:
why not just store these in requests since you have to filter out the noFormKeyRequests on line 318 anyways?